buildbot/osuosl/master/config/builders.py

```diff
@@ -586,8 +586,7 @@
                         "-DCMAKE_CXX_FLAGS='-mcpu=neoverse-v2'",
                         "-DLLVM_ENABLE_LLD=True",
                         "-DMLIR_INCLUDE_INTEGRATION_TESTS=True",
-                        "-DMLIR_RUN_ARM_SVE_TESTS=True",
-                        "-DLLVM_LIT_ARGS='-v'"])},
```

Comment:
[Just checking] Will you remove "-v" from the existing configurations in a separate PR?

Reply:
....I will now that you have brought it up :)

Reply:
https://github.com/llvm/llvm-zorg/pull/322